test/automate/automate-utils.js

```diff
@@ -105,7 +105,7 @@ exports.doesProjectPassTests = function(name, URL) {
 
   // Wait for the item settings modal.
   driver.wait(
-    until.elementLocated(By.id('item-settings-modal')),
+    until.elementLocated(By.css('#item-settings-modal.open')),
     elementTimeout
   );
 
```

Comment:
@tchaffee  Line 112 (after wait for settings modal) was the ONLY place I needed the utils to sleep for two seconds. With that, it ALL passed. You can spare two seconds. :D It's tremendously faster. Awesome!